packages/pwa/app/commerce-api/custom-shopper-baskets.js

```diff
@@ -0,0 +1,16 @@
+import OcapiShopperBaskets from './ocapi-shopper-baskets'
+import * as sdk from 'commerce-sdk-isomorphic'
+
+// This class is an extension to OcapiShopperBaskets class to include basket functions implemented in SCAPI.
+// Current implementation has ShopperBaskets interactiing with OCAPI which misses new functions implemented in newer SCAPI releases.
+class CustomShopperBaskets extends OcapiShopperBaskets {
```

Comment:
🤔 Thinking a bit about naming.

Did you consider calling this `shopper-baskets.js` and class `ShopperBaskets`?

The class is then an implementation of a basket, mixing OCAPI/SCAPI where suitable.

Over time, we expect more of the `ShopperBaskets` implementation to move to SCAPI now that SCAPI hooks shipped ... so this class might be the logical place to continue those extensions, slowly cutting over to SCAPI.

Reply:
Have we considered making the move to remove all our ocapi customizations and getting part of this work for free? I feel like the overall picture we have here is that we are creating an extension to fix a deficiency with the ocapi-shopper-baskets and now that the deficiency is no longer present we are choosing to extend the extension with the built in methods as opposed to just cleaning up our temporary code (aka the ocapi-shopper-basket)

Reply:
So while discussing with Kevin we did consider moving all of ShopperBaskets functions to SCAPI since SCAPI hooks are now released. But that would probably be beyond the scope of this ticket given the amount of work involved.

Reply:
@shethj @kevinxh Do you think it is better if we close this ticket and work on moving all of ShopperBaskets functions to SCAPI instead?

Reply:
Good suggestions. let @shethj and myself do a quick estimation on the work needed to move entire `ShopperBaskets` to use SCAPI. I was a bit worried that doing everything at once is too much and introduces risks. If the work is too large, we'd prefer to make a small diff approach and migrate gradually.

Reply:
@shethj and i are going to make a quick comparison between OCAPI and SCAPI, and answer the following questions:

1. do they have the same set of endpoints
2. are those endpoints return similar data
3. are those endpoints accept similar arguments